src/Utilities/Routes.js

```diff
@@ -38,14 +56,14 @@ export const Routes = () => {
     let path = location.pathname;
     return (
         <Switch>
-            <Redirect from={PATHS.cvepagePath.to} to={PATHS.inventoryDetail.to} />
-            <InsightsRoute exact path={PATHS.cvePage.to} component={CVEPage} />
-            <InsightsRoute exact path={PATHS.inventoryDetail.to}  component={InventoryDetail} />
-            <InsightsRoute exact path={PATHS.vulnerabilitiesCves.to} component={VulnerabilitiesCVEs} />
-            <InsightsRoute exact path={PATHS.systems.to} component={Systems} />
+            <Redirect from={PATHS.cvepagePath.to} to={PATHS.systemDetailsPage.to} />
+            <InsightsRoute exact path={PATHS.cveDetailsPage.to} component={CVEDetailsPage} />
+            <InsightsRoute exact path={PATHS.systemDetailsPage.to}  component={SystemDetailsPage} />
+            <InsightsRoute exact path={PATHS.cvesPage.to} component={LandingPage} />
+            <InsightsRoute exact path={PATHS.systemsPage.to} component={SystemsPage} />
             <InsightsRoute exact path={PATHS.upgrade.to} component={Upgrade} />
             <Route
-                render={() => (some(PATHS, p => p.to === path) ? null : <Redirect to={PATHS.vulnerabilitiesCves.to} />)}
+                render={() => (some(PATHS, p => p.to === path) ? null : <Redirect to={PATHS.cves.to} />)}
```

Comment:
When I run  locally, I have seen that cves.to does not exist anymore and this is causing to crash. It is renamed to `home`

Reply:
update to `cvesPage`